contents/docs/experiments/best-practices.mdx

```diff
@@ -0,0 +1,118 @@
+---
+title: Experiment best practices
+sidebar: Docs
+showTitle: true
+---
+
+## 1. Only include users affected by the change
+
+Including users who aren't affected by the change being tested can lead to unreliable results. For example, if you're testing a new onboarding flow, you don't want to include users who have already completed the flow.
+
+To avoid including unaffected users, make sure to first filter out ineligible users in your code before checking your feature flag code. For example:
+
+```js
+// ✅ Correct. Will exclude unaffected users
+function showNewChanges(user) {
+
+  if (user.hasCompletedAction) {
+    return false
+  }
+
+  // other checks
+
+  if (posthog.getFeatureFlag('experiment-key') === 'control') {
+    return false;
+  }
+
+  return true
+}
+
+// ❌ Incorrect. Will include unaffected users
+function showNewChanges(user) {
+  if (posthog.getFeatureFlag('experiment-key') === 'control') {
+    return false;
+  }
+
+  if (user.hasCompletedAction) {
+    return false
+  }
+
+  // other checks
+
+  return true
+}
+```
+
+## 2. Test your experiment
+
+Sometimes we're so eager to get results from our experiments, we jump straight to running them with all our users. This may be okay if everything is set up correctly, but if you've made a mistake, you may be unable to rerun your experiment. To understand why, let's look at an example:
+
+Imagine you're running an experiment with a 50/50 split between control and test. You roll out the experiment to all your users, but a day after launch, you notice that your change is causing the app to crash for all test users. You immediately stop the experiment and fix the root cause of the crash. However, restarting the experiment now will produce unreliable results since many users have already seen your change.
+
+To avoid this problem, you should first test your experiment with a [small rollout](/tutorials/phased-rollout) (e.g., 5% of users) for a few days. Once you're confident everything works correctly, you can start the experiment with the remaining users.
```

Comment:
How does someone do this in PostHog? Should probably call out they need to edit the feature flag before launching because it is non-obvious.